config/jobs/kubernetes/sig-testing/ec2-e2e.yaml

```diff
@@ -61,6 +61,66 @@ presubmits:
             requests:
               cpu: 8
               memory: 10Gi
+  - name: pull-kubernetes-e2e-ec2-arm64
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+      testgrid-dashboards: presubmits-ec2
+      description: Runs presubmit tests using kubetest2-ec2 against kubernetes master on EC2 (arm64)
+    labels:
+      preset-e2e-containerd-ec2: "true"
+      preset-dind-enabled: "true"
+    path_alias: k8s.io/kubernetes
+    always_run: false
```

Comment:
since those jobs are optional and not run by default, so I feel okay to have those additional jobs, so that anyone want to test it against a pr is possible. 

/lgtm

thanks! @ruquanzhao